actions/signed-commits/src/__snapshots__/run.test.ts.snap

```diff
@@ -34,6 +34,40 @@ exports[`version creates simple PR 1`] = `
 ]
 `;
 
+exports[`version creates simple PR in draft mode with do-not-merge label 1`] = `
```

Comment:
Where is this validating that the label was added to the PR?

Reply:
good question - are we also not just making the
```
  pulls: {
    create: jest.fn(),
  },
```
and just expect it's there?

Reply:
actually I'm going to remove `labels` as we don't need it. I think adding it to the title is perfectly fine:
https://github.com/smartcontractkit/chainlink/pull/12722